cypress/tests/ui/notifications.spec.ts

```diff
@@ -86,7 +86,11 @@ describe("Notifications", function () {
         cy.visit(`/transaction/${transaction.id}`);
       });
 
+      const likesCountSelector = "[data-test*=transaction-like-count]";
```

Comment:
I think this happens several times in this spec file, might as well factor out in a small utility function